.github/workflows/pr.yaml

```diff
@@ -44,18 +58,31 @@ jobs:
     outputs:
       libs: ${{ steps.test-orchestrator.outputs.libs }}
     steps:
-      - name: Member Check
-        if: ${{ github.event.pull_request.author_association != 'MEMBER' }}
+      - name: Testing Check Write Permissions
```

Comment:
There was communication by a Github engineer on the (4 year old!) issue requesting they support yaml anchors in Actions this week.  So hopefully we will soon have a way to not have to repeat the same steps in multiple places.

Reply:
Ah, OK.  I wasn't sure how to handle this.  I'm guessing that isn't ready yet?  So we have to do this repeat step?